sickbeard/webserve.py

```diff
@@ -1661,7 +1654,7 @@ def setStatus(self, show=None, eps=None, status=None, direct=False):
             else:
                 return self._genericMessage("Error", errMsg)
 
-        if not statusStrings.has_key(int(status)):
+        if int(status) not in statusStrings:
```

Comment:
This one has to stay this way, we override has_key for the statusStrings in common.py L544